spec.html

```diff
@@ -149,6 +149,37 @@ <h1>
         1. Return _iteratorRecord_.
       </emu-alg>
     </emu-clause>
+
+    <emu-clause id="sec-getiterator" type="abstract operation">
+      <h1>
+        GetIteratorFlattenable (
+          _obj_: an ECMAScript language value,
+          optional _hint_: ~sync~ or ~async~,
+        ): either a normal completion containing an Iterator Record or a throw completion
+      </h1>
+      <dl class="header">
+      </dl>
+      <emu-alg>
+        1. If _hint_ is not present, set _hint_ to ~sync~.
+        1. If _iterator_ is either *undefined* or *null*, throw a *TypeError* exception.
```

Comment:
This implementation currently allows strings to be returned and treated as iterables. If we wanted to restrict that, we could instead do,

```suggestion
        1. If _iterator_ is not an Object, throw a *TypeError* exception.
```

We could then use `Get` instead of `GetV` on `_obj_` below as well.